plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Fixture.java

```diff
@@ -246,9 +247,15 @@ public static void main(final String[] args) throws Exception {
                         destBucket.objects.put(destObjectName, bytes);
                         return new Response(RestStatus.OK.getStatus(), TEXT_PLAIN_CONTENT_TYPE, EMPTY_BYTE);
                     }
-                }
+                } else {
```

Comment:
I would like us to return an error if `disable_chunked_encoding` is `false` here.

Reply:
Agree, I will submit another commit to fix this.